test/e2e/specs/npm_spec.js

```diff
@@ -36,7 +36,12 @@ describe('OIDC flows', function () {
     primaryAuth.loginToForm('{{{WIDGET_BASIC_USER_3}}}', '{{{WIDGET_BASIC_PASSWORD_3}}}');
     oktaHome.waitForPageLoad();
     Expect.toBeA11yCompliant();
-    expect(oktaHome.getLoggedInUser()).toEqual('Test');
+
+    // userName element is not visible in mobile browsers (have to click a dropdown, not a bug)
+    if (process.env.SAUCE_PLATFORM_NAME === 'iOS') {
+      return;
+    }
+    expect(oktaHome.getLoggedInUser()).toEqual('Test'); // will fail on mobile browsers
```

Comment:
As a follow-up, can we modify the `getLoggedInUser` method to click the dropdown to retrieve this value instead of terminating early from this test?

Reply:
Actually, I just tested this. In the mobile browser, the username is not displayed even in the drop-down. It just shows the `settings` and `signout` links as shown below.
<img width="162" alt="Screen Shot 2019-04-11 at 9 15 55 AM" src="https://user-images.githubusercontent.com/27521424/55973484-8920d680-5c3a-11e9-8b29-0c9d2610943b.png">

Reply:
This is how the username (George) is displayed in desktop browser, which is what we look for - 

<img width="1251" alt="Screen Shot 2019-04-11 at 9 17 38 AM" src="https://user-images.githubusercontent.com/27521424/55973583-bc636580-5c3a-11e9-9570-ae789fc7ba33.png">

Reply:
Gotcha - I misunderstood this comment:

```
// will fail on mobile browsers
```

Can we change it to:

```suggestion
    expect(oktaHome.getLoggedInUser()).toEqual('Test'); // This assertion will always fail because mobile browsers hide the username (expected behavior)
```

Reply:
The element is actually present in the DOM but not visible. Maybe we shall try to adjust the assertion like
- expect the element present and has text `Test`
- element is visible at normal browser but not mobile (smaller screen)

Reply:
Actually, the element doesn't seem to have the text in mobile browser. Here's the error I got on mobile browser - 
```
2) OIDC flows can login and auth in a basic flow
  Message:
    Expected '' to equal 'Test'.
  Stack:
    Error: Failed expectation
      at UserContext.<anonymous> (/home/travis/build/okta/okta-signin-widget/target/e2e/specs/npm_spec.js:39:40)
      at /home/travis/build/okta/okta-signin-widget/node_modules/jasminewd2/index.js:112:25
      at new ManagedPromise (/home/travis/build/okta/okta-signin-widget/node_modules/selenium-webdriver/lib/promise.js:1077:7)
      at ControlFlow.promise (/home/travis/build/okta/okta-signin-widget/node_modules/selenium-webdriver/lib/promise.js:2505:12)
      at schedulerExecute (/home/travis/build/okta/okta-signin-widget/node_modules/jasminewd2/index.js:95:18)
      at TaskQueue.execute_ (/home/travis/build/okta/okta-signin-widget/node_modules/selenium-webdriver/lib/promise.js:3084:14)
      at TaskQueue.executeNext_ (/home/travis/build/okta/okta-signin-widget/node_modules/selenium-webdriver/lib/promise.js:3067:27)
      at asyncRun (/home/travis/build/okta/okta-signin-widget/node_modules/selenium-webdriver/lib/promise.js:2974:25)
      at /home/travis/build/okta/okta-signin-widget/node_modules/selenium-webdriver/lib/promise.js:668:7
      at <anonymous>:null:null
      at process._tickCallback (internal/process/next_tick.js:189:7)
```